apps/backend/src/agents/ProcessingPipeline.ts

```
@@ -1408,6 +1559,37 @@ export const ProcessingPipelineServiceLive = Layer.effect(
                 }
                 break;
 
+              case 'document_links':
+                const allCustomFieldsStream = yield* paperless.getCustomFields();
+                const documentLinkFieldsStream = allCustomFieldsStream.filter((f) => f.data_type === 'documentlink');
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)